app/utils/filters.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const qs = require('qs')
+const moment = require('moment-timezone')
```

Comment:
Just done a bit research on moment-timezone.
The NPM page is saying it is a legacy library and we should move away from it.
I think we should be using the Node Intl library instead.
Also means there is 1 less library we need to maintain.

Reply:
Great, but I hope you know `moment-timezone` is used in multiple places in the project, so its not new.

Reply:
Thanks - I should have seen that 🙈